docs/reference/buildx_dap_build.md

```diff
@@ -0,0 +1,52 @@
+# docker buildx dap build
+
+<!---MARKER_GEN_START-->
+Start a build
+
+### Aliases
+
+`docker build`, `docker builder build`, `docker image build`, `docker buildx b`
+
+### Options
```

Comment:
I'm not sure how these options can be used by editors. Maybe these flags will be need to be moved to the launch config when this DAP plugin allows the editor to invoke the build through the launch request.

Reply:
Thanks for the example. 

> The intention is the debug adapter implementation would translate these to command line arguments. It might be worth putting them into the launch config though just to keep things consistent.

Sure. launch.json support can be following up.